neevaco/neeva-ios
# Fix Other Suggestion Back Button Edge Case

Similar to #3380, just needed the same fix when using the enter key to return from the URL bar.

## Checklists
### How was this tested?

### Manual test cases
- Go to any website from URL bar query search
- Hit the back button
- Type in a new query and tap enter
- Should be on SRP (expected and correct)
